transcom/mymove
# INT B-18753 Move History Log for Service Counselor Approves

## [B-18753](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18753)

## Summary
There are missing details for events logged in Move History where it only shows `-`.
This PR adds in details to missing fields in Move History when SC approve a PPM. 

Changes:

- Removed additional history log created for `mto_shipments` table when SC completes counseling a PPM shipment.
- Shows status `WAITING ON CUSTOMER` in Move History when SC completes counseling a PPM shipment.
- Shows status `PAYMENT APPROVED` in Move History when Closeout SC approves a PPM.

### How to test

1. As a customer -> create a normal move with a PPM shipment.
2. As a SC -> Find the move and complete counseling by making all the edits necessary and clicking submit button. 
3. Go to `Move history` tab and you should see rows with `Counseling Completed` and `Status: WAITING ON CUSTOMER`. Check the uploaded screenshots below for reference.
4. As a customer -> Upload PPM documents and fill out the form to submit to Closeout SC for a review.
5. As a Closeout SC -> Find the move -> Review documents -> Approve all the documents.
6. Go to `Move history` tab and you should see a row with `Status: PAYMENT APPROVED`. Check the uploaded screenshots below for reference.


## Screenshots
### SC completes counseling:
<img width="1319" alt="image" src="https://github.com/transcom/mymove/assets/146856854/4f093d7e-d6fb-4d62-8424-7d79ce4b9d04">

### Closeout SC approves a PPM:
<img width="1313" alt="image" src="https://github.com/transcom/mymove/assets/146856854/f498b5d5-53b4-4104-a7b9-1202b271f5c6">
